[2023-01-07T01:00:43.221+0100] {processor.py:153} INFO - Started process (PID=10943) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:00:43.225+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:00:43.228+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:00:43.227+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:00:43.265+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:00:43.264+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:00:43.265+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:00:43.265+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:00:43.265+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:00:43.283+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.065 seconds
[2023-01-07T01:01:23.656+0100] {processor.py:153} INFO - Started process (PID=10981) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:01:23.657+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:01:23.658+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:01:23.658+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:01:23.674+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:01:23.674+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:01:23.674+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:01:23.674+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:01:23.675+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:01:23.682+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-01-07T01:02:00.317+0100] {processor.py:153} INFO - Started process (PID=11016) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:02:00.318+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:02:00.319+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:02:00.319+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:02:00.334+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:02:00.333+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:02:00.334+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:02:00.334+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:02:00.334+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:02:00.341+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-01-07T01:02:36.817+0100] {processor.py:153} INFO - Started process (PID=11050) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:02:36.819+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:02:36.820+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:02:36.819+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:02:36.839+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:02:36.839+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:02:36.839+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:02:36.839+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:02:36.839+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:02:36.848+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.037 seconds
[2023-01-07T01:03:14.191+0100] {processor.py:153} INFO - Started process (PID=11085) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:03:14.192+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:03:14.193+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:03:14.193+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:03:14.229+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:03:14.228+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:03:14.229+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:03:14.229+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:03:14.229+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:03:14.244+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.064 seconds
[2023-01-07T01:03:50.872+0100] {processor.py:153} INFO - Started process (PID=11119) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:03:50.874+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:03:50.875+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:03:50.874+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:03:50.910+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:03:50.909+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:03:50.910+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:03:50.910+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:03:50.910+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:03:50.926+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.065 seconds
[2023-01-07T01:04:32.173+0100] {processor.py:153} INFO - Started process (PID=11156) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:04:32.174+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:04:32.175+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:04:32.175+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:04:32.190+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:04:32.189+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:04:32.190+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:04:32.190+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:04:32.190+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:04:32.197+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-01-07T01:05:04.276+0100] {processor.py:153} INFO - Started process (PID=11189) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:05:04.279+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:05:04.280+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:05:04.280+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:05:04.308+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:05:04.305+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:05:04.310+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:05:04.309+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:05:04.311+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:05:04.323+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.055 seconds
[2023-01-07T01:05:51.905+0100] {processor.py:153} INFO - Started process (PID=11233) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:05:51.907+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:05:51.907+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:05:51.907+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:05:51.923+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:05:51.923+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:05:51.923+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:05:51.923+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:05:51.924+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:05:51.932+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-01-07T01:06:27.551+0100] {processor.py:153} INFO - Started process (PID=11268) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:06:27.553+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:06:27.555+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:06:27.554+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:06:27.577+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:06:27.576+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:06:27.577+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:06:27.577+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:06:27.577+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:06:27.586+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.040 seconds
[2023-01-07T01:07:01.697+0100] {processor.py:153} INFO - Started process (PID=11301) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:07:01.699+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:07:01.699+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:07:01.699+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:07:01.716+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:07:01.715+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:07:01.716+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:07:01.716+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:07:01.716+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:07:01.723+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-01-07T01:07:36.213+0100] {processor.py:153} INFO - Started process (PID=11333) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:07:36.214+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:07:36.215+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:07:36.215+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:07:36.231+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:07:36.230+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:07:36.231+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:07:36.231+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:07:36.231+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:07:36.239+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-01-07T01:08:19.677+0100] {processor.py:153} INFO - Started process (PID=11369) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:08:19.678+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:08:19.680+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:08:19.679+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:08:19.708+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:08:19.707+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:08:19.708+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:08:19.708+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:08:19.709+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:08:19.723+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.050 seconds
[2023-01-07T01:08:59.472+0100] {processor.py:153} INFO - Started process (PID=11407) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:08:59.473+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:08:59.474+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:08:59.474+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:08:59.490+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:08:59.489+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:08:59.490+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:08:59.490+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:08:59.490+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:08:59.497+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-01-07T01:09:44.124+0100] {processor.py:153} INFO - Started process (PID=11450) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:09:44.135+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:09:44.139+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:09:44.138+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:09:44.190+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:09:44.187+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:09:44.191+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:09:44.191+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:09:44.191+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:09:44.220+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.104 seconds
[2023-01-07T01:10:27.894+0100] {processor.py:153} INFO - Started process (PID=11491) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:10:27.898+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:10:27.899+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:10:27.898+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:10:27.915+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:10:27.914+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:10:27.915+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:10:27.915+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:10:27.915+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:10:27.924+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2023-01-07T01:11:07.684+0100] {processor.py:153} INFO - Started process (PID=11528) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:11:07.685+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:11:07.686+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:11:07.686+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:11:07.700+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:11:07.700+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:11:07.700+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:11:07.700+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:11:07.700+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:11:07.708+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2023-01-07T01:11:47.704+0100] {processor.py:153} INFO - Started process (PID=11566) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:11:47.705+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:11:47.706+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:11:47.706+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:11:47.723+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:11:47.723+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:11:47.723+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:11:47.723+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:11:47.723+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:11:47.731+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-01-07T01:12:27.014+0100] {processor.py:153} INFO - Started process (PID=11603) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:12:27.018+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:12:27.019+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:12:27.019+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:12:27.055+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:12:27.054+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:12:27.055+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:12:27.055+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:12:27.056+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:12:27.068+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.059 seconds
[2023-01-07T01:13:09.350+0100] {processor.py:153} INFO - Started process (PID=11647) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:13:09.352+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:13:09.352+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:13:09.352+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:13:09.368+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:13:09.367+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:13:09.368+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:13:09.368+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:13:09.368+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:13:09.376+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-01-07T01:13:52.524+0100] {processor.py:153} INFO - Started process (PID=11688) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:13:52.526+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:13:52.527+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:13:52.526+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:13:52.544+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:13:52.544+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:13:52.544+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:13:52.544+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:13:52.544+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:13:52.552+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2023-01-07T01:14:41.766+0100] {processor.py:153} INFO - Started process (PID=11730) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:14:41.769+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:14:41.770+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:14:41.770+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:14:41.787+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:14:41.787+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:14:41.787+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:14:41.787+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:14:41.788+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:14:41.795+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2023-01-07T01:15:27.752+0100] {processor.py:153} INFO - Started process (PID=11773) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:15:27.753+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:15:27.754+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:15:27.754+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:15:27.772+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:15:27.772+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:15:27.773+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:15:27.773+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:15:27.773+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:15:27.781+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2023-01-07T01:16:05.021+0100] {processor.py:153} INFO - Started process (PID=11808) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:16:05.022+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:16:05.023+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:16:05.023+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:16:05.042+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:16:05.041+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:16:05.042+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:16:05.042+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:16:05.042+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:16:05.050+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.038 seconds
[2023-01-07T01:16:48.047+0100] {processor.py:153} INFO - Started process (PID=11858) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:16:48.049+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:16:48.050+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:16:48.049+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:16:48.064+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:16:48.064+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:16:48.064+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:16:48.064+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:16:48.065+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:16:48.072+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-01-07T01:17:30.148+0100] {processor.py:153} INFO - Started process (PID=11898) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:17:30.149+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:17:30.149+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:17:30.149+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:17:30.165+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:17:30.164+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:17:30.165+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:17:30.165+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:17:30.165+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:17:30.173+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-01-07T01:18:16.267+0100] {processor.py:153} INFO - Started process (PID=11937) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:18:16.268+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:18:16.269+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:18:16.269+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:18:16.285+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:18:16.284+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:18:16.285+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:18:16.285+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:18:16.285+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:18:16.294+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-01-07T01:19:05.665+0100] {processor.py:153} INFO - Started process (PID=11979) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:19:05.666+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:19:05.667+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:19:05.667+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:19:05.683+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:19:05.683+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:19:05.684+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:19:05.684+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:19:05.684+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:19:05.695+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2023-01-07T01:19:59.597+0100] {processor.py:153} INFO - Started process (PID=12023) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:19:59.598+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:19:59.599+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:19:59.599+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:19:59.615+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:19:59.615+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:19:59.615+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:19:59.615+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:19:59.615+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:19:59.623+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-01-07T01:20:44.542+0100] {processor.py:153} INFO - Started process (PID=12065) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:20:44.543+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:20:44.544+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:20:44.543+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:20:44.560+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:20:44.560+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:20:44.560+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:20:44.560+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:20:44.561+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:20:44.569+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-01-07T01:21:20.108+0100] {processor.py:153} INFO - Started process (PID=12102) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:21:20.110+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:21:20.111+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:21:20.110+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:21:20.131+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:21:20.130+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:21:20.131+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:21:20.131+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:21:20.131+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:21:20.150+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.045 seconds
[2023-01-07T01:21:59.059+0100] {processor.py:153} INFO - Started process (PID=12139) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:21:59.070+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:21:59.071+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:21:59.071+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:21:59.120+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:21:59.120+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:21:59.121+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:21:59.121+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:21:59.121+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:21:59.152+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.119 seconds
[2023-01-07T01:22:42.424+0100] {processor.py:153} INFO - Started process (PID=12177) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:22:42.425+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:22:42.427+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:22:42.426+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:22:42.441+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:22:42.441+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:22:42.442+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:22:42.441+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:22:42.442+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:22:42.449+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-01-07T01:23:24.900+0100] {processor.py:153} INFO - Started process (PID=12218) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:23:24.902+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:23:24.903+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:23:24.902+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:23:24.919+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:23:24.919+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:23:24.919+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:23:24.919+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:23:24.919+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:23:24.927+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-01-07T01:24:07.506+0100] {processor.py:153} INFO - Started process (PID=12256) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:24:07.507+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:24:07.508+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:24:07.508+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:24:07.523+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:24:07.523+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:24:07.523+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:24:07.523+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:24:07.524+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:24:07.531+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2023-01-07T01:24:51.639+0100] {processor.py:153} INFO - Started process (PID=12296) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:24:51.640+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:24:51.641+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:24:51.641+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:24:51.659+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:24:51.658+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:24:51.659+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:24:51.659+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:24:51.659+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:24:51.667+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2023-01-07T01:25:35.740+0100] {processor.py:153} INFO - Started process (PID=12335) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:25:35.741+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:25:35.741+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:25:35.741+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:25:35.758+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:25:35.757+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:25:35.758+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:25:35.758+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:25:35.758+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:25:35.767+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-01-07T01:26:17.146+0100] {processor.py:153} INFO - Started process (PID=12374) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:26:17.147+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:26:17.148+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:26:17.148+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:26:17.164+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:26:17.164+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:26:17.164+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:26:17.164+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:26:17.165+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:26:17.172+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-01-07T01:26:59.584+0100] {processor.py:153} INFO - Started process (PID=12413) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:26:59.585+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:26:59.586+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:26:59.585+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:26:59.601+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:26:59.600+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:26:59.601+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:26:59.601+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:26:59.601+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:26:59.608+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-01-07T01:27:40.652+0100] {processor.py:153} INFO - Started process (PID=12452) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:27:40.653+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:27:40.654+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:27:40.654+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:27:40.668+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:27:40.668+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:27:40.668+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:27:40.668+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:27:40.668+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:27:40.675+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-01-07T01:28:23.216+0100] {processor.py:153} INFO - Started process (PID=12493) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:28:23.218+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:28:23.219+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:28:23.219+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:28:23.235+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:28:23.235+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:28:23.235+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:28:23.235+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:28:23.235+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:28:23.243+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-01-07T01:29:06.813+0100] {processor.py:153} INFO - Started process (PID=12606) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:29:06.814+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:29:06.815+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:29:06.815+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:29:06.833+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:29:06.833+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:29:06.833+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:29:06.833+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:29:06.834+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:29:06.842+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2023-01-07T01:29:49.840+0100] {processor.py:153} INFO - Started process (PID=12689) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:29:49.841+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:29:49.841+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:29:49.841+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:29:49.856+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:29:49.856+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:29:49.856+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:29:49.856+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:29:49.856+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:29:49.863+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2023-01-07T01:30:31.675+0100] {processor.py:153} INFO - Started process (PID=12728) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:30:31.678+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:30:31.679+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:30:31.678+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:30:31.693+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:30:31.693+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:30:31.693+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:30:31.693+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:30:31.694+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:30:31.701+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-01-07T01:31:07.096+0100] {processor.py:153} INFO - Started process (PID=12763) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:31:07.097+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:31:07.098+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:31:07.098+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:31:07.118+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:31:07.117+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:31:07.118+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:31:07.118+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:31:07.118+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:31:07.125+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2023-01-07T01:31:45.668+0100] {processor.py:153} INFO - Started process (PID=12799) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:31:45.669+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:31:45.670+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:31:45.670+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:31:45.684+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:31:45.684+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:31:45.684+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:31:45.684+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:31:45.684+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:31:45.692+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2023-01-07T01:32:26.054+0100] {processor.py:153} INFO - Started process (PID=12837) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:32:26.057+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:32:26.058+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:32:26.058+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:32:26.078+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:32:26.078+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:32:26.079+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:32:26.079+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:32:26.079+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:32:26.092+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.045 seconds
[2023-01-07T01:33:09.060+0100] {processor.py:153} INFO - Started process (PID=12874) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:33:09.061+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:33:09.062+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:33:09.062+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:33:09.078+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:33:09.078+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:33:09.078+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:33:09.078+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:33:09.079+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:33:09.085+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-01-07T01:33:56.109+0100] {processor.py:153} INFO - Started process (PID=12914) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:33:56.110+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:33:56.111+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:33:56.111+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:33:56.130+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:33:56.129+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:33:56.130+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:33:56.130+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:33:56.130+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:33:56.137+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2023-01-07T01:35:01.280+0100] {processor.py:153} INFO - Started process (PID=12966) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:35:01.282+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:35:01.282+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:35:01.282+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:35:01.297+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:35:01.296+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:35:01.297+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:35:01.297+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:35:01.298+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:35:01.306+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-01-07T01:35:47.847+0100] {processor.py:153} INFO - Started process (PID=13010) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:35:47.849+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:35:47.849+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:35:47.849+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:35:47.865+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:35:47.864+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:35:47.865+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:35:47.865+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:35:47.865+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:35:47.873+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-01-07T01:36:31.380+0100] {processor.py:153} INFO - Started process (PID=13050) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:36:31.382+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:36:31.383+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:36:31.383+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:36:31.401+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:36:31.400+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:36:31.401+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:36:31.401+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:36:31.401+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:36:31.409+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2023-01-07T01:37:10.049+0100] {processor.py:153} INFO - Started process (PID=13086) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:37:10.051+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:37:10.052+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:37:10.051+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:37:10.070+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:37:10.069+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:37:10.070+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:37:10.070+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:37:10.070+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:37:10.078+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2023-01-07T01:37:46.942+0100] {processor.py:153} INFO - Started process (PID=13122) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:37:46.944+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:37:46.944+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:37:46.944+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:37:46.961+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:37:46.960+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:37:46.961+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:37:46.961+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:37:46.961+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:37:46.969+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-01-07T01:38:28.598+0100] {processor.py:153} INFO - Started process (PID=13159) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:38:28.599+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:38:28.600+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:38:28.600+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:38:28.615+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:38:28.615+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:38:28.615+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:38:28.615+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:38:28.615+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:38:28.624+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-01-07T01:39:11.562+0100] {processor.py:153} INFO - Started process (PID=13198) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:39:11.564+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:39:11.565+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:39:11.564+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:39:11.581+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:39:11.581+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:39:11.582+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:39:11.582+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:39:11.582+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:39:11.589+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-01-07T01:40:00.302+0100] {processor.py:153} INFO - Started process (PID=13238) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:40:00.304+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:40:00.305+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:40:00.304+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:40:00.335+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:40:00.334+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:40:00.335+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:40:00.335+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:40:00.336+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:40:00.350+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.051 seconds
[2023-01-07T01:40:47.377+0100] {processor.py:153} INFO - Started process (PID=13281) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:40:47.379+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:40:47.380+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:40:47.380+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:40:47.399+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:40:47.399+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:40:47.400+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:40:47.399+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:40:47.400+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:40:47.407+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2023-01-07T01:41:35.146+0100] {processor.py:153} INFO - Started process (PID=13323) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:41:35.148+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:41:35.148+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:41:35.148+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:41:35.163+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:41:35.163+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:41:35.164+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:41:35.164+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:41:35.164+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:41:35.171+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-01-07T01:42:28.871+0100] {processor.py:153} INFO - Started process (PID=13368) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:42:28.872+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:42:28.873+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:42:28.873+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:42:28.893+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:42:28.892+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:42:28.894+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:42:28.893+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:42:28.894+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:42:28.902+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2023-01-07T01:43:21.208+0100] {processor.py:153} INFO - Started process (PID=13413) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:43:21.209+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:43:21.210+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:43:21.210+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:43:21.224+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:43:21.224+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:43:21.224+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:43:21.224+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:43:21.224+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:43:21.231+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-01-07T01:44:07.918+0100] {processor.py:153} INFO - Started process (PID=13453) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:44:07.919+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:44:07.920+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:44:07.920+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:44:07.951+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:44:07.950+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:44:07.951+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:44:07.951+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:44:07.951+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:44:07.966+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.051 seconds
[2023-01-07T01:44:51.515+0100] {processor.py:153} INFO - Started process (PID=13493) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:44:51.516+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:44:51.517+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:44:51.517+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:44:51.531+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:44:51.530+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:44:51.531+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:44:51.531+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:44:51.531+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:44:51.538+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2023-01-07T01:45:34.793+0100] {processor.py:153} INFO - Started process (PID=13531) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:45:34.800+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:45:34.801+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:45:34.801+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:45:34.842+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:45:34.841+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:45:34.842+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:45:34.842+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:45:34.843+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:45:34.852+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.074 seconds
[2023-01-07T01:46:22.586+0100] {processor.py:153} INFO - Started process (PID=13573) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:46:22.588+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:46:22.589+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:46:22.589+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:46:22.607+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:46:22.606+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:46:22.607+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:46:22.607+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:46:22.607+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:46:22.614+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-01-07T01:47:03.247+0100] {processor.py:153} INFO - Started process (PID=13610) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:47:03.248+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:47:03.248+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:47:03.248+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:47:03.264+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:47:03.264+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:47:03.264+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:47:03.264+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:47:03.265+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:47:03.273+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-01-07T01:47:42.557+0100] {processor.py:153} INFO - Started process (PID=13648) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:47:42.560+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:47:42.561+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:47:42.561+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:47:42.581+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:47:42.581+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:47:42.582+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:47:42.582+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:47:42.582+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:47:42.639+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.093 seconds
[2023-01-07T01:48:21.315+0100] {processor.py:153} INFO - Started process (PID=13685) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:48:21.317+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:48:21.318+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:48:21.317+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:48:21.334+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:48:21.334+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:48:21.335+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:48:21.334+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:48:21.335+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:48:21.343+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2023-01-07T01:49:08.405+0100] {processor.py:153} INFO - Started process (PID=13725) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:49:08.407+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:49:08.407+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:49:08.407+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:49:08.424+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:49:08.424+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:49:08.425+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:49:08.425+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:49:08.425+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:49:08.432+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-01-07T01:50:07.213+0100] {processor.py:153} INFO - Started process (PID=13773) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:50:07.215+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:50:07.216+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:50:07.215+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:50:07.231+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:50:07.230+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:50:07.231+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:50:07.231+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:50:07.231+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:50:07.239+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-01-07T01:50:56.920+0100] {processor.py:153} INFO - Started process (PID=13814) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:50:56.930+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:50:56.931+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:50:56.931+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:50:56.997+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:50:56.995+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:50:56.998+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:50:56.998+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:50:56.998+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:50:57.021+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.104 seconds
[2023-01-07T01:52:01.694+0100] {processor.py:153} INFO - Started process (PID=13869) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:52:01.696+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:52:01.697+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:52:01.696+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:52:01.713+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:52:01.712+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:52:01.713+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:52:01.713+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:52:01.713+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:52:01.722+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-01-07T01:52:48.286+0100] {processor.py:153} INFO - Started process (PID=13911) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:52:48.288+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:52:48.288+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:52:48.288+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:52:48.303+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:52:48.303+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:52:48.303+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:52:48.303+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:52:48.303+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:52:48.311+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-01-07T01:53:25.185+0100] {processor.py:153} INFO - Started process (PID=13948) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:53:25.187+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:53:25.188+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:53:25.188+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:53:25.210+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:53:25.209+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:53:25.210+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:53:25.210+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:53:25.210+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:53:25.241+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.060 seconds
[2023-01-07T01:54:04.760+0100] {processor.py:153} INFO - Started process (PID=13984) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:54:04.762+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:54:04.763+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:54:04.763+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:54:04.780+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:54:04.780+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:54:04.781+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:54:04.780+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:54:04.781+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:54:04.788+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2023-01-07T01:54:46.413+0100] {processor.py:153} INFO - Started process (PID=14025) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:54:46.415+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:54:46.416+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:54:46.415+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:54:46.435+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:54:46.435+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:54:46.436+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:54:46.436+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:54:46.436+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:54:46.445+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.038 seconds
[2023-01-07T01:55:30.236+0100] {processor.py:153} INFO - Started process (PID=14063) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:55:30.237+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:55:30.238+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:55:30.238+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:55:30.256+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:55:30.256+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:55:30.257+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:55:30.256+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:55:30.257+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:55:30.265+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2023-01-07T01:56:12.958+0100] {processor.py:153} INFO - Started process (PID=14104) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:56:12.960+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:56:12.961+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:56:12.961+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:56:12.976+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:56:12.976+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:56:12.976+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:56:12.976+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:56:12.977+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:56:12.984+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-01-07T01:56:52.850+0100] {processor.py:153} INFO - Started process (PID=14143) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:56:52.851+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:56:52.852+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:56:52.852+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:56:52.885+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:56:52.885+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:56:52.886+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:56:52.885+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:56:52.886+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:56:52.900+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.062 seconds
[2023-01-07T01:57:54.539+0100] {processor.py:153} INFO - Started process (PID=14189) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:57:54.540+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:57:54.541+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:57:54.541+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:57:54.564+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:57:54.564+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:57:54.565+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:57:54.564+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:57:54.565+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:57:54.577+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.042 seconds
[2023-01-07T01:58:42.848+0100] {processor.py:153} INFO - Started process (PID=14229) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:58:42.850+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:58:42.851+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:58:42.851+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:58:42.901+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:58:42.901+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:58:42.901+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:58:42.901+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:58:42.903+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:58:42.931+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.094 seconds
[2023-01-07T01:59:29.158+0100] {processor.py:153} INFO - Started process (PID=14271) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:59:29.159+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T01:59:29.160+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:59:29.160+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:59:29.179+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:59:29.178+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T01:59:29.179+0100] {logging_mixin.py:137} INFO - [2023-01-07T01:59:29.179+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T01:59:29.179+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T01:59:29.189+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2023-01-07T02:00:16.050+0100] {processor.py:153} INFO - Started process (PID=14319) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:00:16.051+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T02:00:16.052+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:00:16.052+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:00:16.069+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:00:16.068+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T02:00:16.069+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:00:16.069+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T02:00:16.069+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:00:16.077+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-01-07T02:01:03.721+0100] {processor.py:153} INFO - Started process (PID=14364) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:01:03.722+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T02:01:03.723+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:01:03.723+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:01:03.738+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:01:03.738+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T02:01:03.738+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:01:03.738+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T02:01:03.738+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:01:03.747+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-01-07T02:01:43.658+0100] {processor.py:153} INFO - Started process (PID=14400) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:01:43.659+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T02:01:43.660+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:01:43.660+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:01:43.675+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:01:43.675+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T02:01:43.675+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:01:43.675+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T02:01:43.676+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:01:43.683+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-01-07T02:02:31.326+0100] {processor.py:153} INFO - Started process (PID=14441) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:02:31.327+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T02:02:31.327+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:02:31.327+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:02:31.341+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:02:31.341+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T02:02:31.341+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:02:31.341+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T02:02:31.341+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:02:31.348+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-01-07T02:03:17.262+0100] {processor.py:153} INFO - Started process (PID=14481) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:03:17.265+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T02:03:17.269+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:03:17.269+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:03:17.306+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:03:17.306+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T02:03:17.306+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:03:17.306+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T02:03:17.307+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:03:17.328+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.074 seconds
[2023-01-07T02:04:02.271+0100] {processor.py:153} INFO - Started process (PID=14523) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:04:02.272+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T02:04:02.273+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:04:02.272+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:04:02.289+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:04:02.289+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T02:04:02.289+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:04:02.289+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T02:04:02.290+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:04:02.298+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-01-07T02:04:42.273+0100] {processor.py:153} INFO - Started process (PID=14560) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:04:42.274+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T02:04:42.275+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:04:42.274+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:04:42.290+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:04:42.290+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T02:04:42.290+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:04:42.290+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T02:04:42.290+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:04:42.298+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-01-07T02:05:25.757+0100] {processor.py:153} INFO - Started process (PID=14598) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:05:25.759+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T02:05:25.760+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:05:25.759+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:05:25.777+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:05:25.777+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T02:05:25.778+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:05:25.778+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T02:05:25.778+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:05:25.785+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2023-01-07T02:06:04.089+0100] {processor.py:153} INFO - Started process (PID=14632) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:06:04.090+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T02:06:04.091+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:06:04.090+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:06:04.111+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:06:04.110+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T02:06:04.111+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:06:04.111+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T02:06:04.111+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:06:04.132+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.046 seconds
[2023-01-07T02:06:41.158+0100] {processor.py:153} INFO - Started process (PID=14663) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:06:41.161+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T02:06:41.162+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:06:41.162+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:06:41.184+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:06:41.183+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T02:06:41.184+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:06:41.184+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T02:06:41.184+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:06:41.197+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.045 seconds
[2023-01-07T02:07:31.405+0100] {processor.py:153} INFO - Started process (PID=14705) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:07:31.406+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T02:07:31.407+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:07:31.407+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:07:31.425+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:07:31.424+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T02:07:31.425+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:07:31.425+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T02:07:31.425+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:07:31.432+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-01-07T02:08:23.756+0100] {processor.py:153} INFO - Started process (PID=14748) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:08:23.770+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T02:08:23.771+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:08:23.771+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:08:23.794+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:08:23.794+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T02:08:23.795+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:08:23.794+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T02:08:23.795+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:08:23.831+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.079 seconds
[2023-01-07T02:09:20.566+0100] {processor.py:153} INFO - Started process (PID=14795) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:09:20.568+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T02:09:20.568+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:09:20.568+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:09:20.585+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:09:20.585+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T02:09:20.585+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:09:20.585+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T02:09:20.586+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:09:20.593+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-01-07T02:10:06.103+0100] {processor.py:153} INFO - Started process (PID=14836) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:10:06.105+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T02:10:06.106+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:10:06.106+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:10:06.126+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:10:06.126+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T02:10:06.126+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:10:06.126+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T02:10:06.126+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:10:06.133+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2023-01-07T02:10:52.515+0100] {processor.py:153} INFO - Started process (PID=14876) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:10:52.517+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T02:10:52.518+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:10:52.518+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:10:52.536+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:10:52.536+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T02:10:52.536+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:10:52.536+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T02:10:52.537+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:10:52.544+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2023-01-07T02:11:36.296+0100] {processor.py:153} INFO - Started process (PID=14919) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:11:36.297+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T02:11:36.298+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:11:36.298+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:11:36.313+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:11:36.312+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T02:11:36.313+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:11:36.313+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T02:11:36.313+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:11:36.320+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-01-07T02:12:22.335+0100] {processor.py:153} INFO - Started process (PID=14958) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:12:22.338+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-07T02:12:22.340+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:12:22.340+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:12:22.362+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:12:22.362+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-07T02:12:22.362+0100] {logging_mixin.py:137} INFO - [2023-01-07T02:12:22.362+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-07T02:12:22.363+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-07T02:12:22.379+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.050 seconds
